pallets/asset-manager/src/benchmarking.rs

```diff
@@ -47,21 +47,17 @@ benchmarks! {
 		let start = <T::AssetConfig as AssetConfig<T>>::StartNonNativeAssetId::get();
 		let end = start + 1000;
 		for i in start..end {
-
 			let location: MultiLocation = MultiLocation::new(0, X1(Parachain(i)));
 			let location = <T::AssetConfig as AssetConfig<T>>::AssetLocation::from(location.clone());
 			let metadata = <T::AssetConfig as AssetConfig<T>>::AssetRegistrarMetadata::default();
-
 			Pallet::<T>::register_asset(RawOrigin::Root.into(), location.clone(), metadata.clone())?;
 			Pallet::<T>::set_units_per_second(RawOrigin::Root.into(), i, 0)?;
 		}
-
 		// does not really matter what we register, as long as it is different than the previous
 		let location = <T::AssetConfig as AssetConfig<T>>::AssetLocation::default();
 		let metadata = <T::AssetConfig as AssetConfig<T>>::AssetRegistrarMetadata::default();
 		let amount = 10;
-		Pallet::<T>::register_asset(RawOrigin::Root.into(), location.clone(), metadata.clone())?;
```

Comment:
Btw all of these clones() in the setup and verification are not part the benchmark itself, so they didn't impact the performance.